Integrations/integration-Alexa.yml

```diff
@@ -0,0 +1,119 @@
+description: Alexa provides website ranking information that can be useful in determining
+  if the domain in question has a strong web presence.
+detaileddescription: To use this integration, simply call ```!awis-info```. !awis
+  accepts "url" as a parameter to query.
+configuration:
+- display: Sensitivity threshold for configuring which domains are suspicious versus
+    trusted.
+  name: threshold
+  defaultvalue: "2000000"
+  type: 0
+  required: true
+script:
+  script: |
+    import urllib, sys, re
+    import xml.etree.ElementTree as ET
+    from collections import defaultdict
+
+
+    """GLOBAL VARIABLES/CONSTANTS"""
+    THRESHOLD = int(demisto.params().get('threshold'))
+
+
+    """MAIN FUNCTIONS"""
+    def awis(domain):
+        xml = urllib.urlopen('http://data.alexa.com/data?cli=10&dat=s&url=%s'%domain).read()
+        root = ET.fromstring(xml)
+        return root
+
+
+    """COMMAND FUNCTIONS"""
+    def awis_command():
+        domain = demisto.args().get('domain')
+        root = awis(domain)
+        dbot_ec = defaultdict()
+        dom_ec = defaultdict()
+        # Wrapping in Try Except since when no results are found, endpoint returns 404
+        try:
+            domain_res = root.find("SD").attrib['HOST']
+            rank = root.find("SD[0]/POPULARITY").attrib['TEXT']
+            if int(rank) > THRESHOLD:
+                dbot_score = 2
+            else:
+                dbot_score = 0
+        except:
+            rank = 'Unknown'
+            dbot_score = 2
+        dom_ec['Name'] = domain
+        dbot_ec['Score'] = dbot_score
+        dbot_ec['Vendor'] = 'Alexa'
+        dbot_ec['Rank'] = rank
+        dbot_ec['Domain'] = domain
+        ec = {
+            'Domain(val.Name && val.Name == obj.Name)': dom_ec,
+            'DBotScore': dbot_ec
+        }
+        demisto.results({
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['markdown'],
+                    'Contents': 'The Alexa rank of {} is {}'.format(domain, rank),
+                    'HumanReadable': 'The Alexa rank of {} is {}'.format(domain, rank),
```

Comment:
I would add `marked as suspicous/good/bad domain` or just add `while the threshold is X`